public/assets/js/spongedownloads.js

```diff
@@ -0,0 +1,388 @@
+const API = "https://dev.minecrell.net/downloads"
```

Comment:
@progwml6 Added `DOWNLOADS_API_URL`, must be set at build time (when `gulp` is executed) and will default to Sponge's URL (right now it points to my demo server)